df.military.xml

```diff
@@ -258,7 +258,7 @@
 
             <vmethod is-destructor='true'/>
             <vmethod name='offsetPosition'>
-                <int16_t name='x'/> <int16_t name='y'/> <int16_t name='z'/>
```

Comment:
Did you intentionally get rid of the vmethod parameter names in this file?

Reply:
No, definitely not.  I'm not aware of having touched them at all.

Reply:
Thanks for addressing - there are also two other instances below in this file (I should have highlighted the whole thing, oops).

Reply:
Weird. I can only assume it's automatic merging that's somehow been done backwards.

Reply:
Yeah, Git won't be able to tell the difference between changes you made to a file and changes you copied in from an older version of the file (i.e. pulling won't help because it thinks all of the changes in your version are "newer").
The diff I was looking at that showed their removal was https://github.com/DFHack/df-structures/pull/392/files (the "Files" tab above). It looks good now!